content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3558,14 +3559,20 @@ curl -X POST \
 
 <Text>
 
-### List Registered Webhooks
+### List Webhooks
 
 #### Specifics
 
 |                 |                                                                                  |
 | --------------- | -------------------------------------------------------------------------------- |
-| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/list_registered_webhooks` |
-| Required scopes | `webhooks--my:ro` `webhooks--all:ro`                                             |
+| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/list_webhooks`            |
+| Required scopes | `webhooks.configuration:rw`                                                      |
+
+#### Request
+
+| Parameter         | Required | Data type | Notes                                                                                        |
+| ----------------- | -------- | --------- | -------------------------------------------------------------------------------------------- |
+| `owner_client_id` | no       | `string`  | ClientID of desired webhook owner. Required when Personal Access Token is used to authorize. |
```

Comment:
```suggestion
| `owner_client_id` | no       | `string`  | The webhook owner (the Client Id for which the webhook is registered). It's required when a Personal Access Token is used for authorization. |
```

Reply:
The webhook(s) is already registered, so it has an owner (that's why I used the Present tense instead of the Future tense).